Models/studentModels/select.ts

```diff
@@ -0,0 +1,9 @@
+import { executeQuery } from '../../utils/database';
+import { StudentProps } from '../../Interfaces/studentinterface';
+
+export const studentSelect = async (): Promise<StudentProps[]> => {
+    const selectAllStudents = "SELECT * FROM `Student`";
```

Comment:
Sequelize:
Sequelize is a promise-based Node.js ORM for Postgres, MySQL, MariaDB, SQLite, and Microsoft SQL Server. It features solid transaction support, relations, read replication, and more.

Knex.js:
Knex.js is a "batteries included" SQL query builder for PostgreSQL, MySQL, SQLite3, and Oracle. It can help you build complex queries with a clean and readable syntax.